config/txloader/load/harvestcraft/lang/pl_PL.lang

```diff
@@ -0,0 +1,1136 @@
+item.limeyogurtItem.name=Limonkowy Yogurt
+        
+item.mangojuiceItem.name=Sok z Mango
+item.mangosmoothieItem.name=Smoothie z Mango
+item.mangoyogurtItem.name=Jogurt z Mango
+        
+item.pomegranatejuiceItem.name=Sok z Granatu
+item.pomegranatesmoothieItem.name=Smoothie z Granatu
+item.pomegranateyogurtItem.name=Jogurt z Granatu
+        
+item.vanillayogurtItem.name=Waniliowy Jogurt
+        
+item.cinnamonrollItem.name=Rolada Cynamonowa
+item.frenchtoastItem.name=Tost Francuski
+        
+item.marshmellowsItem.name=Pianki
+item.donutItem.name=Donut
+item.chocolatedonutItem.name=Czekoladowy Donut
+item.powdereddonutItem.name=Donut z Cukrem Pudrem
+item.jellydonutItem.name=Donut z Dżemem
+item.frosteddonutItem.name=Donut z Polewą
+        
+item.cactussoupItem.name=Zupa Kaktusowa
+item.wafflesItem.name=Gofry
+item.seedsoupItem.name=Zupa z Nasion
+item.softpretzelItem.name=Precel
+item.jellybeansItem.name=Żelki Fasolki
+item.biscuitItem.name=Ciastko
+        
+item.creamcookieItem.name=Ciastko z Kremem
+item.jaffaItem.name=Delicja
+item.friedchickenItem.name=Smażony Kurczak
+item.chocolatesprinklecakeItem.name=Czekoladowe Ciasto z Posypką
+item.redvelvetcakeItem.name=Ciasto Red Velvet
+item.footlongItem.name=30cm Kanapka
+        
+item.blueberryyogurtItem.name=Jagodowy Jogurt
+item.lemonyogurtItem.name=Cytrynowy Jogurt
+item.cherryyogurtItem.name=Wiśniowy Jogurt
+item.strawberryyogurtItem.name=Truskawkowy Jogurt
+item.grapeyogurtItem.name=Winogronowy Jogurt
+item.chocolateyogurtItem.name=Czekoladowy Jogurt
+        
+item.blackberryjuiceItem.name=Sok Jeżynowy
+item.blackberrycobblerItem.name=Jeżynowy Placek
+item.blackberrysmoothieItem.name=Jeżynowe Smoothie
+item.blackberryyogurtItem.name=Jeżynowy Jogurt
+item.chocolatemilkItem.name=Mleko Czekoladowe
+item.pumpkinyogurtItem.name=Jogurt z Dyni
+        
+item.raspberryjuiceItem.name=Malinowy Sok
+item.raspberrypieItem.name=Ciasto Malinowe
+item.raspberrysmoothieItem.name=Malinowe Smoothie
+item.raspberryyogurtItem.name=Malinowy Jogurt
+item.cinnamonsugardonutItem.name=Donut z Cukrem i Cynamonem
+item.melonyogurtItem.name=Arbuzowy Jogurt
+        
+item.kiwijuiceItem.name=Sok z Kiwi
+item.kiwismoothieItem.name=Smoothie z Kiwi
+item.kiwiyogurtItem.name=Jogurt z Kiwi
+item.plainyogurtItem.name=Jogurt Naturalny
+item.appleyogurtItem.name=Jabłkowy Jogurt
+        
+item.saltedsunflowerseedsItem.name=Solone Nasiona Słonecznika
+item.sunflowerwheatrollsItem.name=Bułka Pszenna ze Słonecznikiem
+item.sunflowerbroccolisaladItem.name=Sałatka z Brokuła i Słonecznika
+        
+item.cranberryjuiceItem.name=Sok Żurawinowy
+item.cranberrysauceItem.name=Sos Żurawinowy
+item.cranberrybarItem.name=Ciastko Żurawinowe
+        
+item.peppermintItem.name=Miętówka
+        
+item.cactusfruitjuiceItem.name=Sok z Opuncji
+        
+item.blackpepperItem.name=Czarny Pieprz
+item.groundcinnamonItem.name=Mielony Cynamon
+item.groundnutmegItem.name=Mielona Gałka Muszkatołowa
+item.oliveoilItem.name=Olej Kuchenny
```

Comment:
Oliwa z oliwek? olej kuchenny to chyba inny item, ale jeżeli nie i ten item wytwarza się z czegoś więcej niż z oliwek to tłumaczenie jest ok

Reply:
to czemu go autor nazwał oliwa z oliwek lmao

Reply:
W en_US.lang jest taka linijka, pewnie autor zmienił nazwę a wewn. nazwa została :)
`item.oliveoilItem.name=Cooking Oil`